### PR TITLE
docs/Windows: More cleanup

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

File: docs/WindowsBuild.md [hunk 1]
```diff
@@ -126,8 +126,8 @@ set llvm_bin_dir="%swift_source_dir%/build/Ninja-DebugAssert/llvm-windows-amd64/
 - You may need to adjust the `SWIFT_WINDOWS_LIB_DIRECTORY` parameter depending on
   your target platform or Windows SDK version.
 ```cmd
-mkdir "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
-pushd "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
```
Comment (AndrewSB) [inline docs/WindowsBuild.md]: @compnerd I can't remember if one needs to `cd` into the ninja dir to run cmake. Is `pushd`ing into the parent ok? 
If you're not sure I'll probably run this Monday/Tuesday next week and we can test then

Comment (compnerd) [reply docs/WindowsBuild.md]: Id say test, but you can pass the path to the cmake invocation.

Comment (AndrewSB) [reply docs/WindowsBuild.md]: I'll test on the next time I do a clean run

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Comment by RUSshy** [comment]
Is there a place where i can get nightly build ?

**Comment by RUSshy** [comment]
yes i mean snapshot, building is too much to get started, why there is no such thing yet? it will make it easier for everyone to come and help with testing and bug reports

**Pull request merged**